src/kartikeyDJune20/AllDifferIndexInStringArray.java

```diff
@@ -0,0 +1,41 @@
+package kartikeyDJune20;
+
+public class AllDifferIndexInStringArray {
+	 static void checkDifference(String[]input1, String[] input2) {
+
+		if (input1.length == input2.length) {
+
+			boolean flag = true;
+			int index = 0;
+			for (; index < input1.length; index++) {
+
+				if (!input1[index].equals(input2[index]) ) {
+					flag = false;
+					//System.out.println("Values are not matching at index: " + index);
+					
+					System.out.println("Values are not matching at index: " + index + " First Array values is: "+input1[index]+" second array value is: "+input2[index]);
+
+				}
+			}
+			if (flag == true) {
```

Comment:
No need of this logic. Because here you are printing AllDifferIndexInStringArray().

Reply:
Changes incorporated... Thanku